Extensions/Firebase/JsExtension.js

```diff
@@ -1,4 +1,7 @@
 // @flow
+/*::
+import { type Trans } from '../../newIDE/app/node_modules/@lingui/macro'
```

Comment:
I just noticed you didn't do that for the other extensions. Is it bad?

Reply:
@arthuro555 This typing is invalid, it should be `(string) => string`. See how it's done in other extensions. Trans is a React like component, not a function.